ChangeLog.md

```diff
@@ -50,6 +50,9 @@
 - *DEC*
   - Adding missing headers in some files of DEC.
     (Roland Denis, [#1349](https://github.com/DGtal-team/DGtal/pull/1349))
+    
+- *Math*
+  - Fix possible division by zero in the MultiStatistics class.
```

Comment:
```suggestion
  - Fix possible division by zero in the MultiStatistics class. (Kacper Pluta, [#1358](https://github.com/DGtal-team/DGtal/pull/1358))
```

Reply:
could you please apply my suggestion ? thx

Reply:
done